src/controller.cpp

```diff
@@ -124,6 +129,32 @@ int Controller::run(const CliArgsParser& args)
 		return args.return_code().value();
 	}
 
+	if (!can_open(configpaths.url_file())) {
+		if (!args.silent()) {
+			std::cout << strprintf::fmt(_("Starting %s %s..."),
+					PROGRAM_NAME,
+					utils::program_version())
+				<< std::endl;
+
+			std::cout << '\n';
```

Comment:
Maybe this gets removed anyways, but is the mix of `'\n'` and `std:endl` here intentional? Couldn't be the first `std::endl` a `'\n'` as well, because the `std::endl` down below will flush stdout?

Reply:
[This will be removed](https://github.com/newsboat/newsboat/pull/907#discussion_r414707165).